x-pack/plugins/lens/public/indexpattern_datasource/indexpattern.tsx

```diff
@@ -248,17 +248,27 @@ export function getIndexPatternDatasource({
       render(
         <KibanaThemeProvider theme$={core.theme.theme$}>
           <I18nProvider>
-            <IndexPatternDataPanel
-              data={data}
-              dataViews={dataViews}
-              fieldFormats={fieldFormats}
-              charts={charts}
-              indexPatternFieldEditor={dataViewFieldEditor}
-              {...otherProps}
-              core={core}
-              uiActions={uiActions}
-              onIndexPatternRefresh={onRefreshIndexPattern}
-            />
+            <KibanaContextProvider
```

Comment:
I think wrapping the field list component could also prevent the need for this since we could add it to `LensAppServices` and access it with `useKibana<LensAppServices>().services.unifiedFieldList.FieldList`.